bin/check-puppet-last-run.rb

```diff
@@ -95,7 +107,11 @@ def run
       @message += " with #{@failures} failures"
     end
 
-    if @now - @last_run > config[:crit_age] || @failures > 0
+    if @restart_failures > 0
```

Comment:
I see, guess since I was not super familiar with this code base and felt an explicit check made sense that way I would not have to try think of edge cases here.